pywbem/tupletree.py

```diff
@@ -41,9 +41,9 @@
 
 The NAME is the name of the element.
 
-The ATTRS are a name-value hash of element attributes.
+The ATTRS are a name-value dictionary of element attributes, not pres. order.
```

Comment:
what does pres. mean

Reply:
ahha.  preserving.  Lets spell it out.

Reply:
Was too long for the line. But you are right, we should spell it out.

Reply:
DONE